app/presenters/finder_presenter.rb

```diff
@@ -125,7 +125,7 @@ def display_key_for_metadata_key(key)
   end
 
   def atom_url
-    ["#{slug}.atom", values.to_query].reject(&:blank?).join("?")
+    ["#{slug}.atom", values.to_query].reject(&:blank?).join("?") unless default_order
```

Comment:
Since the condition is non-intuitive, I think it would be better to pull this into a private `#atom_feed_enabled?` method. That way the `#atom_url` only has to worry about how to build the URL, instead of whether it should build it at all.